### PR TITLE
Custom crawls

### PR DESCRIPTION
**Pull request by b5**
Sprint prep checklist. This PR should:
* Update repo to new package names (`archive` -> `core`, etc.)
* Add stub `README.md`, moving old readme to readme_prev.md if it existed
* Add `.github` stuff: CONTRIBUTING, ISSUES
* Add support for codecov.io to CI tests
* confirm existence of a `LICENSE` file

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,15 +1,14 @@
-# datatogether json api
+# Sentry
```
Comment (dcwalk) [inline readme.md]: Again wrong readme

**Review by dcwalk — CHANGES_REQUESTED** [review]
Looks okay excepting readme, do you want to add back in godeps?

**Comment by dcwalk** [comment]
This PR should address #4   and https://github.com/datatogether/roadmap/issues/53

**Comment by titaniumbones** [comment]
@b5 I was about to work on this but looks like you're in progress. Should I merge or do you want to add the godeps back in?

**Comment by b5** [comment]
hey sorry! I'm touching lots of repos, feel free to attack this guy!

**Pull request merged**